app/src/main/java/framgia/com/mynote/data/model/CallBack.java

```diff
@@ -0,0 +1,9 @@
+package framgia.com.mynote.data.model;
+
+import java.util.List;
+
+public interface CallBack {
+    void onGetDataSuccess(List<Note> notes);
+
+    void onGetDataFailed();
```

Comment:
```suggestion
    void onGetDataFailed();
```
Trong trường hợp fail ko trả về lỗi cụ thể là gì sao